crates/bevy_tasks/src/task.rs

```diff
@@ -12,20 +15,58 @@ use core::{
 /// more gracefully and wait until it stops running, use the [`Task::cancel()`] method.
 ///
 /// Tasks that panic get immediately canceled. Awaiting a canceled task also causes a panic.
-#[derive(Debug)]
```

Comment:
This is an API surface regression. Either keep the derive, or manually implement `Debug`.

Reply:
There's a manual debug impl lower down in the file. Derives don't work on type macros.